commons/helpers/pods_helper.py

```diff
@@ -504,6 +504,20 @@ def kill_process_in_container(self, pod_name, container_name, process_name):
         option = '-9'
         cmd = commands.PKIL_CMD.format(option+" "+process_name)
         resp = self.send_k8s_cmd(operation="exec", pod=pod_name, namespace=const.NAMESPACE,
-                                command_suffix=f"-c {container_name} -- {cmd}",
-                                decode=True)
+                                 command_suffix=f"-c {container_name} -- {cmd}",
+                                 decode=True)
         return resp
+
+    def get_all_container_processes(self, pod_name, container_name):
+        """
+        Function to get all processes running on given container
```

Comment:
@ApurwaMohite We can add better description to this. 
As this lists all the processes from consul, listed processes are not specific to given container.